src/System.Net.Requests/tests/AssemblyInfo.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true, MaxParallelThreads = 1)]
```

Comment:
>Ok. Please add a comment indicating why parallelism is being disabled. Eventually we'd want to re-enable it. Does disabling parallelism have a measurable impact on run time of the tests?

We cannot do that when these tests are run against NETFX.  It is "by-design" that they hang on NETFX if the default value for ServicePointManager.DefaultConnectionLimit is left as 2. We can perhaps change that limit when running the tests against NETFX.  The problem is that when the tests are run in parallel, it leaves too many connections opern at once which causes the subsequent tests to hang due to not being able to get a new connection.